package.json

```diff
@@ -15,9 +15,11 @@
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
     "react-hot-toast": "^2.4.1",
+    "react-markdown": "^9.0.1",
     "react-router-dom": "^7.0.2",
     "react-simple-star-rating": "^5.1.7",
-    "react-star-ratings": "^2.3.0"
+    "react-star-ratings": "^2.3.0",
+    "remark-gfm": "^4.0.0"
```

Comment:
What are these needed for?

Reply:
star ratings was present previously, remark-gfm is for github flavored markdown